address_book/manifest.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: v1
+items:
+  - apiVersion: v1
+    kind: Service
+    metadata:
+      annotations:
+        kompose.cmd: kompose convert -f docker-compose.yml -o manifest.yaml
+        kompose.version: 1.24.0 (4a2a0458)
+      creationTimestamp: null
+      labels:
+        io.kompose.service: api
+      name: api
+    spec:
+      ports:
+        - name: "8080"
+          port: 8080
+          targetPort: 8080
+      selector:
+        io.kompose.service: api
+    status:
+      loadBalancer: {}
+  - apiVersion: v1
```

Comment:
Why are we exposing database?

Reply:
Do you mean adding service for DB?

it needs for communication db and service inside the cluster via internal url like svc.cluster.local

Reply:
I thought we need LoadBalancers for external, my bad.